backend/packages/Upgrade/src/api/controllers/ExcludeController.ts

```diff
@@ -110,34 +108,40 @@ export class ExcludeController {
    *           schema:
    *             type: object
    *             required:
-   *               - id
+   *               - userIds
    *             properties:
-   *               id:
-   *                type: string
-   *           description: UserId to exclude
+   *               userIds:
+   *                type: array
+   *                items:
+   *                  type: string
+   *           description: UserIds to exclude globally
    *       tags:
    *         - Exclude
    *       produces:
    *         - application/json
    *       responses:
    *          '200':
-   *            description: Exclude user from experiment
+   *            description: Exclude user globally
    *            schema:
    *              $ref: '#/definitions/userExcludeResponse'
+   *          '401':
+   *            description: Authorization Required Error
+   *          '500':
+   *            description: Internal Server Error, Insert Error in database, JSON format is not valid
    */
-  @Put('/user')
-  public excludeUser(@BodyParam('id') id: string): Promise<ExplicitIndividualExclusion> {
-    return this.exclude.excludeUser(id);
+  @Post('/user')
+  public excludeUser(@BodyParam('userIds') userIds: Array<string>): Promise<ExplicitIndividualExclusion[]> {
+    return this.exclude.excludeUser(userIds);
```

Comment:
@VivekFitkariwala Can we change the function name as well? Like `excludeUsers`. As its not about a single user from now on.

Reply:
+1